### ISSUE TITLE
setup.exe file runs application but does not install it

### ISSUE THREAD

**Issue by benjtinsley** [electron/windows-installer#242]
somewhat similar to #240 and #218 when running the make through electron forge task, it compiles and exports the app with no problem. when i go to install the app on a windows environment, it shows the installation gif and the app is opened, but never installed. when i close the app, it is not installed on my system so there is no reference of it. is this expected behavior or have i misconfigured something that causes this? the documentation seems like setup.exe should install it but i may be misinterpreting this.

i am running windows installer on electron forge.

my configuration:

```
"electronWinstallerConfig": {
  "appDirectory": "./out/Desktop-win32-ia32",
  "name": "Desktop",
  "outputDirectory": "./out/make/desktop-windows",
  "setupExe": "DesktopSetup.exe",
  "certificateFile": "./certificates/certificate.p12"
}
```

**Comment by shawkinsl** [comment]
Running into this myself. I feel like this has to be "didn't read docs" or something...

**Comment by shawkinsl** [comment]
aha! I think I found it: it looks like the setup.exe "installs" to `%APPDATA%/../Local/yourapp` . Weird.

**Comment by positlabs** [comment]
This is a pretty big issue for me. I've tried the example code, just requiring electron electron-squirrel-startup and exiting doesn't seem to do anything that it promises. No shortcut icon and no menu icon. Even searching for the app in the windows menu doesn't bring it up.

**Comment by ltomov** [comment]
I actually gave up on the squirrel installer and used nullsoft’s nsis.

It does everything properly.

**Comment by alexanderturinske** [comment]
Electron saves its data in `%APPDATA%/../Local/yourapp` on Windows by default (`~/Library/Application\ Support/yourapp` on Mac). I do not think you can change that.

**Comment by positlabs** [comment]
Even if it does install there, why isn't `electron-squirrel-startup` installing the shortcut / menu icons? How is the user supposed to know to look in that directory?

**Comment by dfeinzeig** [comment]
any update on this? doesn't seem like setup actually does much setup...

**Comment by DarinDev1000** [comment]
Is there a way to install start menu shortcuts when it installs?

**Comment by Rejjak** [comment]
yes, I have the same question, I need to shortcut file of my electron app

**Comment by atlanteavila** [comment]
I'm facing the same problem. Any idea on a fix?

**Comment by wslp12** [comment]
any one ?

**Comment by ifelsegrandmaster** [comment]
I have an idea, why not add a shortcut when the application starts up.

**Comment by Unknown807** [comment]
I was thinking just package your exe in the generated zip file using InnoSetup, because though its installed to %appdata%, theres no shortcuts or anything.
